gc3libs/backends/batch.py

```diff
@@ -153,6 +154,7 @@ def __init__(self, name,
                 username=self._username,
                 port=auth.port,
                 keyfile=(keyfile or auth.keyfile),
+                pkey=auth.pkey,
```

Comment:
If you take `pkey` as a parameter in the constructor, it should act as an override here:
```suggestion
                pkey=(pkey or auth.pkey),
```
If there's no foreseen use for overrides, please remove the additional parameter from the constructor.

Reply:
Fixed in 651f88cd9fa2ae19d9f8a1b6cfeab76b718f5b2e